src/include/ipc/header-intel-cavs.h

```diff
@@ -0,0 +1,80 @@
+/* SPDX-License-Identifier: BSD-3-Clause
+ *
+ * Copyright(c) 2019 Intel Corporation. All rights reserved.
+ *
+ * Author: Liam Girdwood <liam.r.girdwood@linux.intel.com>
+ *         Keyon Jie <yang.jie@linux.intel.com>
+ */
+
+/**
+ * \file include/ipc/header-intel-cavs.h
+ * \brief IPC command header for Intel cAVS platforms
+ * \author Liam Girdwood <liam.r.girdwood@linux.intel.com>
+ * \author Keyon Jie <yang.jie@linux.intel.com>
+ */
+
+/* primary register
+ * DIPCTDR in sideband IPC (cAVS 1.8+)
+ * DIPCT in cAVS 1.5 IPC
+ */
+
+#define CAVS_IPC_RSVD_31		BIT(31)
```

Comment:
Are all these IPCs the micro IPCs ? do they duplicate any of the general IPCs ?

Reply:
There are general defs at the beginning followed by module message specific defs (below the `/* Module messages */ ....`). I used qualifiers: `CAVS_IPC_...`, `CAVS_IPC_MOD_...`, `CAVS_IPC_MOD_SETD0IX_...` to indicate levels in hierarchy. Perhaps double `_` could help to separate the qualifier and name of the bit field? More documentation and distribution to headers dedicated for each message class are coming as well as soon as we agree on how the translation cavs_msg->sof_msg is handled.